privacyidea/models.py

```diff
@@ -1461,6 +1490,26 @@ def get(self, key=None):
             ret = d
         return ret
 
+
+class PolicyCondition(MethodsMixin, db.Model):
+    __tablename__ = "policycondition"
+
+    id = db.Column(db.Integer, Sequence("policycondition_seq"), primary_key=True)
+    policy_id = db.Column(db.Integer, db.ForeignKey('policy.id'), nullable=False)
+    section = db.Column(db.Unicode(255), nullable=False)
+    key = db.Column(db.Unicode(255), nullable=False)
```

Comment:
In order to prevent conflicts with Oracle, we shouldn't use lower-case ``key`` as a column name.

Reply:
done in e5951e406cca3de9c09f261c0e16d1920fbaeaa6